src/context/mod.rs

```diff
@@ -550,9 +550,16 @@ pub trait Context<'a>: ContextInternal<'a> {
     }
 
     #[cfg(all(feature = "napi-4", feature = "event-queue-api"))]
-    /// Creates an unbounded queue of events to be executed on a JavaScript thread
-    fn queue(&mut self) -> EventQueue {
-        EventQueue::new(self)
+    /// Creates an unbounded channel for scheduling events to be executed on the JavaScript thread.
+    fn channel(&mut self) -> Channel {
+        Channel::new(self)
+    }
+
+    #[cfg(all(feature = "napi-4", feature = "event-queue-api"))]
+    #[deprecated(since = "0.9.0", note = "Please use the channel() method instead")]
```

Comment:
@kjvalencik Is 0.9.0 the right version to choose here? I was guessing that would be the next version we ship.

Reply:
Yes, we can ship that next. 👍  I'll keep that in mind for the next release.